### PR TITLE
Add support for new calculation mode on partial refund

### PR DESCRIPTION
**Pull request by mntzrr**

### CODE REVIEW

File: classes/requests/helpers/class-pb2b-order-lines.php [hunk 1]
```diff
@@ -88,7 +88,7 @@ public static function get_product_name( $order_item ) {
 	 */
 	public static function get_line_unit_price( $order_item ) {
 		$quantity      = 0 === abs( $order_item->get_quantity() ) ? 1 : abs( $order_item->get_quantity() );
-		$item_subtotal = round( ( $order_item->get_total() ) / $quantity * 100, 2 );
+		$item_subtotal = abs( round( ( $order_item->get_total() ) / $quantity * 100, 2 ) );
```
Comment (MichaelBengtsson) [inline classes/requests/helpers/class-pb2b-order-lines.php:91]: @montazarr Why was this needed? Just so that i know

Comment (mntzrr) [reply classes/requests/helpers/class-pb2b-order-lines.php:91]: @MichaelBengtsson the new endpoint expects a positive number. It fails otherwise.

**Review by MichaelBengtsson — APPROVED** [review]
